tests/unit_tests/utils/test_proof_of_contribution.py

```diff
@@ -0,0 +1,28 @@
+import os
+from unittest.mock import Mock
+
+import pytest
+from pytest import MonkeyPatch
+
+from chatgpt.protocol import ValidationMessage
+from chatgpt.utils.proof_of_contribution import proof_of_contribution
+
+
+@pytest.fixture
+def file_zip():
+    return os.path.join(os.getcwd(), 'tests/data/chatgpt_1_conversation.zip')
+
+
+@pytest.mark.asyncio
+async def test_proof_of_contribution(mocker: MonkeyPatch, file_zip: str) -> None:
+    # Mock the decryption function and return our mocked file
+    mock_decrypt: Mock = mocker.patch('chatgpt.utils.proof_of_contribution.download_and_decrypt_file')
+    mock_decrypt.return_value = file_zip
+    mock_os: Mock = mocker.patch('chatgpt.utils.proof_of_contribution.os.remove')
```

Comment:
Should we also mock the OpenAI API?

Reply:
This small unit test was meant to test the entire validation flow after decrypting the file, so I chose not to mock OpenAI. But I gave an example of mocking so if someone wanted to mock OpenAI, it'd be easy to do.